package.json

```diff
@@ -12,17 +12,21 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.6.1",
+    "@types/react-plotly.js": "^2.5.0",
     "plotly.js-dist": "^2.2.0",
-    "react-graph-vis": "^1.0.5",
+    "react-grid-layout": "^1.2.5",
     "react-plotly.js": "^2.5.1"
-  },
+  }
   "devDependencies": {
+    "@types/react-grid-layout": "^1.1.2",
     "@types/enzyme-adapter-react-16": "^1.0.6",
     "@types/react-plotly.js": "^2.2.4",
     "@types/react-test-renderer": "^16.9.1",
     "cypress": "^5.0.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "eslint": "^6.8.0",
     "jest-dom": "^4.0.0"
+    "react-graph-vis": "^1.0.5",
```

Comment:
why is this moved to `devDependencies`?

Reply:
This got created while resolving merge conflicts. Will move it back.